src/main/java/Duke/todoList/task/task.java

```diff
@@ -0,0 +1,18 @@
+package Duke.todoList.task;
+
+import Duke.Duke;
+
+public class task {
```

Comment:
The [class name conventions ](https://se-education.org/guides/conventions/java/basic.html#layout)could apply here.